src/data/Docente.ts

```diff
@@ -0,0 +1,18 @@
+import { Usuario } from "./Usuario";
+
+export class Docente extends Usuario {
+    constructor (
+        protected especialidade: string[],
```

Comment:
se a propriedade é uma array, é importante colocar o nome da variável no plural: especialidades. Isso ajuda muito no entendimento do código, quando essa variável for utilizada longe da declaração (onde se vê de que tipo ela é)